### ISSUE TITLE
Autoreplay not working on exim 4.94-1

### ISSUE THREAD

**Issue by moucho** [madeITBelgium/vesta#86]
### Operating System (OS/VERSION):

CentOS 7

### VestaCP Version:
0.0.24

### Installed Software (what you got with the installer):

apache, nginx, mysql, exim

### Steps to Reproduce:

Setting an autoreplay for a mail account doesn't work, shows an error in exim log:

xxx@xxx.com R=autoreplay T=userautoreply defer (13): Permission denied: Tainted '/etc/exim/domains/xxx/autoreply.info.msg' (file for userautoreply transport) not permitted

### Related Issues/Forum Threads:

Because the changes in exim 4.94-1

**Comment by moucho** [comment]
@madeITBelgium I've tested the solution on https://github.com/madeITBelgium/vesta/commit/f39058f03045c043e01bdd03c22ecf18930a583a but it's not working either.

I think "userautoreply" transport should be modify also. I've tried with no avail so far.

**Comment by moucho** [comment]
Hi,
I've tried the latest fix, sadly it's not working:

```
2020-10-23 15:44:02 1kVxMQ-0000Qb-Rk ** info@example.com R=autoreplay T=userautoreply: Expansion of "${lookup{$local_part}dsearch{${extract{5}{:}{${lookup{$local_part}lsearch{/etc/exim/domains/${lookup{$domain}dsearch{/etc/exim/domains/}}/passwd}}}}/mail/${lookup{$domain}dsearch{/etc/exim/domains/}}}}@${lookup{$domain}dsearch{/etc/exim/domains/}}" failed in userautoreply transport: failed to open /etc/exim/domains/example.com/passwd for linear search: Permission denied (euid=93 egid=93)
```
I checked file permissions but didn't fix it

**Comment by madeITBelgium** [comment]
Fixed in 0.0.36: https://github.com/madeITBelgium/vesta/commit/e79122a5f8eaf23defe688e784e602960f2e5f0d
